File: jna/src/main/java/dark/dxrep.java
package dark;

import com.sun.jna.Pointer;
import com.sun.jna.Structure;

import java.util.Arrays;
import java.util.List;

public class dxrep extends Structure {
    public float dt, db, dl, dr;

    public dxrep() {
        super();
    }

    public dxrep(Pointer peer) {
        super(peer);
    }

    @Override
    protected List<String> getFieldOrder() {
        return Arrays.asList("dt","db","dl","dr");
    }
}

File: jna/src/main/java/dark_kotlin/dxrep.kt
package com.sl.microservice.ai.runtime.impl.darknet.dark

import com.sun.jna.Pointer
import com.sun.jna.Structure

class dxrep : Structure {
    @JvmField val dt = 0f
    @JvmField val db = 0f
    @JvmField val dl = 0f
    @JvmField val dr = 0f

    constructor() : super()
    constructor(peer: Pointer) : super(peer)

    override fun getFieldOrder(): List<String> {
        return listOf("dt", "db", "dl", "dr")
    }
}
